Tests/PointFreeTests/AuthTests.swift

```diff
@@ -17,30 +17,30 @@ class AuthTests: TestCase {
       |> \.gitHubUser.id .~ 1234567890
       |> \.gitHubUser.name .~ "Blobby McBlob"
 
-    let env: (Environment) -> Environment =
-      (\.database .~ .live)
-        <> (\.gitHub.fetchUser .~ const(pure(gitHubUserEnvelope.gitHubUser)))
-        <> (\.gitHub.fetchAuthToken .~ const(pure(pure(gitHubUserEnvelope.accessToken))))
-
-    AppEnvironment.with(env) {
-      let result = connection(
-        from: request(to: .gitHubCallback(code: "deabeef", redirect: "/"), session: .loggedOut)
-        )
-        |> siteMiddleware
-        |> Prelude.perform
-      assertSnapshot(matching: result)
-
-      let registeredUser = AppEnvironment.current.database
-        .fetchUserByGitHub(gitHubUserEnvelope.gitHubUser.id)
-        .run
-        .perform()
-        .right!!
-
-      XCTAssertEqual(gitHubUserEnvelope.accessToken.accessToken, registeredUser.gitHubAccessToken)
-      XCTAssertEqual(gitHubUserEnvelope.gitHubUser.id, registeredUser.gitHubUserId)
-      XCTAssertEqual(gitHubUserEnvelope.gitHubUser.name, registeredUser.name)
-      XCTAssertEqual(1, registeredUser.episodeCreditCount)
-    }
+    update(
+      &Current,
+      \.database .~ .live,
+      \.gitHub.fetchUser .~ const(pure(gitHubUserEnvelope.gitHubUser)),
+      \.gitHub.fetchAuthToken .~ const(pure(pure(gitHubUserEnvelope.accessToken)))
```

Comment:
I have you found that `update(&Current, ...)` type checks better than how we were doing `env` previously?

Reply:
It's about the same :/